Doc improvements (#589)
pyansys/pyaedt: doc/source/API/Setup.rst

Setup
=====
This section lists setup modules:
* Setup for ``Hfss``, ``Maxwell2D``, ``Maxwell3d``, ``Q2d`` and ``Q3d``
* SetupCircuit for ``Circuit`` and ``Simplorer``
* Setup3DLayout for ``Hfss3dLayout``
Setup object is accessible through the ``create_setup`` method and ``setups`` object list.
.. code:: python

    from pyaedt import Hfss
    app = Hfss(specified_version="2021.1",
                 non_graphical=False, new_desktop_session=True,
                 close_on_exit=True, student_version=False)

>     # this call return the Setup Class

Reviewer: ```suggestion
    # this call returns the Setup Class
```

    my_setup = app.setups[0]

>     # this call return a Setup Object

Reviewer: ```suggestion
    # this call returns a Setup Object
```

=====
This section lists setup modules:

.. currentmodule:: pyaedt.modules.SolveSetup

.. autosummary::
   :toctree: _autosummary
   :template: custom-class-template.rst
   :nosignatures:

   Setup
   Setup3DLayout
   SetupCircuit

Sweep Classes
=============

> This section lists the Sweeos classes and their default values.

Reviewer 2: ```suggestion
This section lists Sweep classes and their default values.
```
